physics/mirror_formulae.py

```diff
@@ -57,13 +57,13 @@
 
 """
 
-
+from math import isclose
```

Comment:
Please move this import into the doctest so that the linter does not get confused.